.github/workflows/compile-and-test-latest.yml

```diff
@@ -0,0 +1,26 @@
+name: compile-and-test-latest
```

Comment:
The action did not run as part of the PR, but it ran on my fork here:
https://github.com/kevinAlbs/mongo-cxx-driver/actions/runs/521114465

Reply:
Nice